<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>520</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>780</width>
    <height>520</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>780</width>
    <height>520</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-10</y>
      <width>781</width>
      <height>531</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/new/prefix1/resou/res_img/scenery_bg2.jpg);</string>
    </property>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>250</y>
       <width>120</width>
       <height>135</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="title">
      <string>监控</string>
     </property>
     <widget class="QPushButton" name="monitoring_btn">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>25</y>
        <width>100</width>
        <height>100</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/new/prefix1/resou/res_img/Icon/monitoring_jpg.jpg);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>70</y>
       <width>120</width>
       <height>135</height>
      </rect>
     </property>
     <property name="title">
      <string>音乐播放器</string>
     </property>
     <widget class="QPushButton" name="music_btn">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>100</width>
        <height>100</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/new/prefix1/resou/res_img/Icon/music_pic.jpg);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>70</y>
       <width>120</width>
       <height>135</height>
      </rect>
     </property>
     <property name="title">
      <string>智能家居</string>
     </property>
     <widget class="QPushButton" name="smart_home_btn">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>25</y>
        <width>100</width>
        <height>100</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/new/prefix1/resou/res_img/samrt_home_pic.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>250</y>
       <width>120</width>
       <height>135</height>
      </rect>
     </property>
     <property name="title">
      <string>天气预报</string>
     </property>
     <widget class="QPushButton" name="weather_mainbtn">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>25</y>
        <width>100</width>
        <height>100</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/new/prefix1/resou/res_img/Icon/sun_Gif.gif);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>780</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
